packages/share_movie_plugin/lib/const.dart

```diff
@@ -0,0 +1,4 @@
+class ShareMoviePluginAndroid {
+  ShareMoviePluginAndroid._();
+  static const intentTitle = 'Movie Sharing';
```

Comment:
pass it as a param to the plugin. since in another app u will have other title, not Movie Sharing